hail/python/hail/backend/service_backend.py

```diff
@@ -320,6 +324,11 @@ def __init__(self,
         self.worker_memory = worker_memory
         self.name_prefix = name_prefix
         self.regions = regions
+        self._file_scheme_cache = []
+        self._storage_location_allow_list = gcs_bucket_allow_list
```

Comment:
Is this used? I couldn't find `_storage_location_allow_list` anywhere.

Reply:
removed